docs/repository.md

```diff
@@ -16,3 +16,25 @@ Worked example of branch structure:
 * **1.1** - contains the latest deployed patch release of OCDS 1.1 (e.g. 1.1.0)
   * **1.1-dev** - used to stage changes to version 1.1 (such as minor documentation changes)
   * **1.1.1-dev** - used to work on a 1.1.1 version (including schema changes and fixes)
+
+## Repository structure
+
+* [.tx/config](https://github.com/open-contracting/standard/blob/1.1-dev/.tx/config) - config file for transifex
```

Comment:
> https://github.com/open-contracting/standard/blob/1.1-dev/.tx/config

I wonder if in the standard repository we should have a master branch always even with latest release branch. That way we  would have a URL to link to that we don't need to update

Reply:
It feels like a fair bit of overhead for a small gain. I don't expect we'll have that many links to the latest release branch, and we would only have to change them at most once a year because we don't make minor/major releases frequently.

Reply:
I disagree, I don't think merging in now and then (once a year if you want, as nothing else should be happening in master) it is as much overhead as having to change every single link _at least_ on this page, once a year. Also, this is quite likely to be forgotten and left behind.

Reply:
Hmm, I was figuring we'd want to actually keep the branch up-to-date. It seems unnecessarily confusing to link to an out-of-date branch.

Reply:
Personally, I don't think the links in this file are adding a dramatic amount of value. We can just leave out the links if they are causing a problem. The value of this file is to orient a person who (in all likelihood) has a local copy of the files, and just needs some sign-posts to find their way around.

Reply:
It seems we can also replace `1.1-dev` with `HEAD` without having to create a branch, e.g. https://github.com/open-contracting/standard/blob/HEAD/.tx/config

Reply:
> It seems we can also replace 1.1-dev with HEAD without having to create a branch, e.g. https://github.com/open-contracting/standard/blob/HEAD/.tx/config

Great!